lib/spree_social/engine.rb

```diff
@@ -55,7 +55,7 @@ class Facebook < OAuth2
                             'webos|amoi|novarra|cdm|alcatel|pocket|ipad|iphone|mobileexplorer|' \
                             'mobile'
       def request_phase
-        options[:scope] ||= 'email,offline_access'
+        options[:scope] ||= 'email'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.